app/uk/gov/hmrc/fhregistrationfrontend/views/business_type.scala.html

```diff
@@ -1,92 +1,112 @@
+
 
 @this(
-        mainTemplate: uk.gov.hmrc.fhregistrationfrontend.views.html.main_template_old
+        layout: layout, viewHelpers: ViewHelpers, govukErrorMessage: GovukErrorMessage
 )
 
-@(businessTypeForm: Form[BusinessType], backUrl: String)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@(businessTypeForm: Form[BusinessType], backUrl: String)(implicit request: Request[_], messages: Messages,
+        appConfig: AppConfig)
+
+@pageScripts = {<script type="text/javascript" src='@routes.Assets.at("javascripts/fh.js")'></script>}
 
 @typeNotListed = {
-    <p>@Messages("fh.business_type.details1")</p>
-    <p><a href="@Messages("link.guidance_page")" target="_blank">@Messages("fh.business_type.detailsLinkText")</a> @Messages("fh.business_type.details2")</p>
+    <details class="govuk-details" data-module="govuk-details">
+        <summary class="govuk-details__summary">
+            <span class="govuk-details__summary-text">
+            @Messages("fh.business_type.summary")
+            </span>
+        </summary>
+        <div class="govuk-details__text">
+            <p>@Messages("fh.business_type.details1")</p>
+            <a class="govuk-link" href="@Messages("link.guidance_page")" target="_blank">@Messages("fh.business_type.detailsLinkText")</a> @Messages("fh.business_type.details2")</p>
+        </div>
+    </details>
 }
 
-@scripts = {<script type="text/javascript" src='@routes.Assets.at("javascripts/fh.js")'></script>}
-
 @titlePrefix = @{
     if (businessTypeForm.hasErrors) Messages("fh.generic.errorPrefix") + " " else ""
 }
 
-@title = @{
-    Messages("fh.business_type.title")
-
+@pageTitle = @{
+    Messages("fh.business_type.title") + " - " + Messages("service.name") + " - " +
+            Messages("gov.uk")
 }
 
-@mainTemplate(title = titlePrefix + title, pageScripts = Some(scripts)) {
-
-    <link rel="stylesheet" href='@routes.Assets.at("stylesheets/fh.css")'/>
-    <a href="@backUrl" class="link-back">@Messages("fh.generic.back")</a>
-    @error_box(businessTypeForm)
-
-    @helper.form(action = uk.gov.hmrc.fhregistrationfrontend.controllers.routes.Application.submitBusinessType, 'novalidate -> "novalidate") {
-
-        @helper.CSRF.formField
-
-        @RadioGroup(
-            RadioGroupParams(
-                name = Some("businessType"),
-                inline = false,
-                legend = Some(title),
-                legendAsHeading = true,
-                error = Some(Messages("fh.businessType.error.required")),
-                hasErrors = businessTypeForm("businessType").hasErrors,
-                value = businessTypeForm("businessType").value,
-                radios = List(
-                    RadioParams(
-                        "CorporateBody",
-                        Messages("fh.business_type.limited_company")
+
+@layout(pageTitle, Some(pageScripts), appConfig) {
+
+    <a href="@backUrl" class="govuk-back-link">@Messages("fh.generic.back")</a>
```

Comment:
you have to use govukBackLink component

Reply:
It's not available yet in this branch.   This currently works.

Reply:
you can add that under viewhelpers and then use govukBackLink

Reply:
If you want to wait for my code and rebase , that's also fine provided that goes before this code :)